### PR TITLE
Use toggle panel from makepad

### PR DESCRIPTION
**Pull request by noxware**
Screenshot 2024-07-31 at 11 58 13

### CODE REVIEW

File: src/shared/toggle_panel.rs [hunk 1]
```diff
@@ -5,116 +5,54 @@ live_design! {
     import makepad_widgets::theme_desktop_dark::*;
```
Comment (noxware) [inline src/shared/toggle_panel.rs:5]: Please ignore this file update.

I can't import this widget from makepad right now so I'm updating the version here in Moxin.
We should delete this file and use makepad's exported panel instead before merging.

File: Cargo.toml [hunk 2]
```diff
@@ -29,7 +27,7 @@ moxin-protocol = { path = "moxin-protocol" }
 moxin-backend = { path = "moxin-backend" }
 moxin-fake-backend = { path = "moxin-fake-backend" }
 
-makepad-widgets = { git = "https://github.com/jmbejar/makepad", branch = "moxin-release-v1" }
```
Comment (noxware) [inline Cargo.toml:32]: We need to revert this before merging.

But cherry picking the toggle panel commits on top of `moxin-release-v1` doesn't work. In that branch, the `import View` inside toggle panel causes a stack overflow (similar to what would happen in the `rik` branch if you try to import something from `base` inside toggle panel.

We may need to update moxin-release-v1 with stuff from rik.

File: Cargo.toml [hunk 3]
```diff
@@ -29,7 +27,7 @@ moxin-protocol = { path = "moxin-protocol" }
...
+makepad-widgets = { git = "https://github.com/noxware/makepad", rev = "7ebad2443" }
```
Comment (noxware) [inline Cargo.toml:30]: This is a jmbejar's fork compatible branch with the widget added

https://github.com/noxware/makepad/tree/moxin-panel-for-jmbejar-remote

Comment (jmbejar) [reply Cargo.toml]: Done!

**Review by jmbejar — APPROVED** [review]

**Pull request merged**